sdk/devcenter/Azure.Developer.DevCenter/README.md

```diff
@@ -36,16 +36,11 @@ To use Azure Active Directory authentication, add the Azure Identity package:
 You will also need to register a new AAD application, or run locally or in an environment with a managed identity.
 If using an application, set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
 
-```
-Uri endpoint = new Uri("<dev-center-uri>");
-var client = new DevCenterClient(endpoint, new DefaultAzureCredential());
-```
-
```

Comment:
Could you please address [this](https://github.com/Azure/azure-sdk-for-net/pull/36245/files#r1320671190) comment?

Reply:
Added one snippet that creates all three clients for our services and then we use them throughout the readme.